src/box/relay.cc

```diff
@@ -1113,11 +1114,10 @@ relay_subscribe(struct replica *replica, struct iostream *io, uint64_t sync,
 	/*
 	 * Save the first vclock as 'received'. Because it was really received.
 	 */
-	vclock_copy(&relay->last_recv_ack.vclock, start_vclock);
+	vclock_copy_ignore0(&relay->last_recv_ack.vclock, start_vclock);
 	relay->r = recovery_new(wal_dir(), false, start_vclock);
-	vclock_copy(&relay->tx.vclock, start_vclock);
+	vclock_copy_ignore0(&relay->tx.vclock, start_vclock);
```

Comment:
Hm, thanks for noticing!

I checked and versions before 1.10.2 don't have a concept of local spaces (firstly introduced here: https://github.com/tarantool/tarantool/commit/f64f46199e19542fa60eede939d62cd115abb83a).

Using `copy_ignore0` just in case I would rather avoid. If I would use it in the first place, I would never find the bug that I am fixing in this PR.